src/main/cpp/Robot.cpp

```diff
@@ -27,25 +31,29 @@ void Robot::TeleopInit() {
 
 void Robot::TeleopPeriodic() {
 
-  double dx = ljoy.GetRawAxis(1);
```

Comment:
The d stood for derivative, so like the derivative of x would be the velocity in the x direction. But since many people haven't taken calculus, I figured it'd be more clear to have a v instead to stand for velocity.